### PR TITLE
fixtures: allow to include global settings for ddb writer;

### PR DESCRIPTION
**Pull request by josketres**
This is needed to be able to create global secondary indexes.

### CODE REVIEW

File: pkg/fixtures/writer_ddb.go [hunk 1]
```diff
@@ -4,22 +4,21 @@ import (
...
-	config  cfg.Config
-	logger  mon.Logger
-	modelId *mdl.ModelId
+	config   cfg.Config
+	logger   mon.Logger
+	settings ddb.Settings
 }
 
-func DynamoDbFixtureWriterFactory(modelId *mdl.ModelId) FixtureWriterFactory {
+func DynamoDbFixtureWriterFactory(settings ddb.Settings) FixtureWriterFactory {
```
Comment (applike-ss) [inline pkg/fixtures/writer_ddb.go]: why not put a pointer here?

File: pkg/fixtures/writer_ddb.go [hunk 2]
```diff
@@ -4,22 +4,21 @@ import (
...
 	"github.com/applike/gosoline/pkg/mon"
 )
 
 type dynamoDbFixtureWriter struct {
-	config  cfg.Config
-	logger  mon.Logger
-	modelId *mdl.ModelId
+	config   cfg.Config
+	logger   mon.Logger
+	settings ddb.Settings
```
Comment (applike-ss) [inline pkg/fixtures/writer_ddb.go]: why not put a pointer here?

File: test/fixtures_dynamodb_test.go [hunk 3]
```diff
@@ -46,17 +48,32 @@ func (s FixturesDynamoDbSuite) TestDynamoDb() {
 	gio, err := s.db.GetItem(&dynamodb.GetItemInput{
 		Key: map[string]*dynamodb.AttributeValue{
 			"Name": {
-				S: mdl.String("Ash"),
+				S: aws.String("Ash"),
 			},
 		},
-		TableName: mdl.String("gosoline-test-integration-test-test-application-testModel"),
+		TableName: aws.String("gosoline-test-integration-test-test-application-testModel"),
```
Comment (applike-ss) [inline test/fixtures_dynamodb_test.go]: +1 style points :)

File: pkg/fixtures/writer_ddb.go (hunk 2 above)
Comment (josketres) [reply pkg/fixtures/writer_ddb.go]: ✅

File: pkg/fixtures/writer_ddb.go (hunk 1 above)
Comment (josketres) [reply pkg/fixtures/writer_ddb.go]: ✅

**Pull request merged**